src/components/UTIL_FormField.component

```diff
@@ -59,7 +55,7 @@
         rendered="{!($ObjectType[sObjType].fields[field].updateable
                     || $ObjectType[sObjType].fields[field].createable && sObj['Id']=='') && $ObjectType[sObjType].fields[field].type != 'boolean'}">        
         <apex:outputLabel for="inputX" styleClass="slds-form-element__label" rendered="{!showLabel == true && ftype!='boolean'}" id="lblInputField">
-            <apex:outputText rendered="{!isRequired}" id="lblInputFieldRequired">
```

Comment:
@jjbennett should there be a comment in the code to let future devs know not to go back to a var for these 2 lines? Otherwise I could see someone refactoring the code to make that same change again.

Reply:
@force2b - @RadSFDeveloper talked about this and we are thinking it might be better to start a known issues doc, there was a similar comment about not using an apex:variable but the component is so large it might easily be missed.

Reply:
Great idea. We can include a note about the two Custom Labels issues as well (LEX and System.Label.).